shared/teams/team/settings/retention/container.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import {connect, compose, lifecycle, type TypedState} from '../../../../util/container'
+import {getTeamRetentionPolicy, makeRetentionPolicy} from '../../../../constants/teams'
+import {type _RetentionPolicy} from '../../../../constants/types/teams'
+import {createGetTeamRetentionPolicy} from '../../../../actions/teams-gen'
+import {navigateAppend} from '../../../../actions/route-tree'
+import type {ConversationIDKey} from '../../../../constants/types/chat2'
+import RetentionPicker from './'
+
+// type 'simple' = we pass up selected values and parent deals with showing warning popup
```

Comment:
lets talk about this. can't it always be auto

Reply:
yeah, it may go away during the info panel changes